src/main/java/com/google/songgame/servlets/GameServlet.java

```diff
@@ -140,6 +142,21 @@ public static YouTube getService() throws GeneralSecurityException, IOException
         .build();
   }
 
+  /**
+   * 
+   * Call to add playlist information to datastore.
```

Comment:
The "Call to" part is redundant and can be removed. Technically you have to call all methods for them to do anything, so you could add this phrase to the start of any method's javadoc no matter what it does, which means it isn't really useful information to add to the javadoc.

Following the guidelines in https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment, this can just be "Adds playlist information to datastore."

Also, you have an extra blank line at the top of the doc which should be removed